metricbeat/docs/modules/tomcat.asciidoc

```diff
@@ -0,0 +1,58 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+
+[[metricbeat-module-tomcat]]
+[role="xpack"]
+== Tomcat module
+
+beta[]
+
+This module periodically fetches JMX metrics from Apache Tomcat.
+
+[float]
+=== Compatibility
+The module has been tested with Tomcat 7.0.24 and 9.0.24. Other versions are expected to work.
+
+[float]
+=== Usage
+Tomcat module requires <<metricbeat-module-jolokia,Jolokia>>to fetch JMX metrics. Refer to the link for instructions about how to use Jolokia.
```

Comment:
```suggestion
The Tomcat module requires <<metricbeat-module-jolokia,Jolokia>>to fetch JMX metrics. Refer to the link for instructions about how to use Jolokia.
```

Reply:
Thanks! Fixed in the last commit. We cannot edit the files under `metricbeat/docs/modules/*` directly because they are generated from the files you can find in `[x-pack/]metricbeat/[module]/_meta/*.asciidoc`. I did the fix there and then a `make update` will update all files in `metricbeat/docs/modules/`